### DISCUSSION TITLE
How can I exclude Stripe? (Not available in current country)

### DISCUSSION THREAD

**Discussion by rasderty01** [vercel/next-forge#358] [Q&A] [answered]
Hi! Thank you for your work on this. I'd like to know which parts I should remove if I want to exclude Stripe? Currently, it's not available in my country (PH). And, I understand that Stripe is needed but just wanted to check if it can be excluded if ever.

**Comment by haydenbleasel** [comment, marked as answer]
Hi @rasderty01, we've tried to isolate as much payments logic to the `packages/payments` package as we can, but there's still some stuff in the apps to deal with like the payments webhook.

Check out the [Lemon Squeezy migration guide](https://docs.next-forge.com/migrations/payments/lemon-squeezy) for an idea of all the files you may need to replace.

**Reply by rasderty01** [reply]
Thank you for your response @haydenbleasel! I did not catch that migration part. I'll take it from here and see if I could set this up with Lemon Squeezy. 😃

**Reply by haydenbleasel** [reply]
Sounds good! If there's any other payments provider you'd prefer, let me know and I can look into writing a migration guide.
